build-system/tasks/extension-helpers.js

```diff
@@ -570,31 +604,49 @@ async function buildExtensionCss(extDir, name, version, options) {
 
   const versions = [version, aliasedVersion].filter(Boolean);
 
-  const bundles = await fastGlob(`${extDir}/*.css`);
+  const cssFiles = await fastGlob(`${extDir}/*.css`);
 
-  await Promise.all(
-    bundles.map(async (filename) => {
-      const name = path.basename(filename, '.css');
-      const css = await jsifyCssAsync(filename);
-      await writeCssBinaries(name, versions, css);
-
-      if (options.bento) {
-        await buildBentoCss(name, versions, css);
-      }
-    })
+  const cssBundles = [];
+  const ampCssBundles = await Promise.all(
+    cssFiles.map(async (filename) => [
+      path.basename(filename, '.css'),
+      await jsifyCssAsync(filename),
+    ])
   );
+
+  cssBundles.push(...ampCssBundles);
+
+  if (options.bento) {
+    /**
+     * For bento, create 1 css bundle for each extension encompassing styles for all components
+     * todo(kvchari): this 1 file can have potentially multiple sourceURL comments. is that fine?
```

Comment:
not fine - breaks sourcemaps. need to think about how to do this because it'll affect what we publish to npm as well. 

Idea: we can create an intermediate file for each bento extension that `@import`s the css files for each component in the extension. On npm, we will need to include all css files for the extension in the npm package for the extension. But this will create extra requests if deployed on the cdn and not ideal.